### DISCUSSION TITLE
5GHz not working?!

### DISCUSSION THREAD

**Discussion by UnleashTheCode** [xchwarze/wifi-pineapple-cloner#14] [Q&A] [answered]
Hello,

Not very experienced with this and just start playing and testing with a GL-AR750S-Ext. One question regarding the Recon module without any adapters only the 2.4GHz APs are recognised in when Both is selected. I tried only 5GHz and no AP showed up. Is this the normal/wanted behaviour?

**Comment by xchwarze** [comment, marked as answer]
The routers do not have dual adapters like the one that comes with TETRA. They have one on X frequency and another on Y. Being two completely separate.
If you want to use it that way you have to do it with an usb adapter.

**Reply by UnleashTheCode** [reply]
Ahh I understand. I have an awus036ach, but I saw is not supported. :( 

Any tips regarding this? If can be supported or the patch or things that need to be done?

**Reply by xchwarze** [reply]
I don't know what chipset that one uses but I have the AWUS036ACM and it works perfectly

**Reply by UnleashTheCode** [reply]
RTL8812AU, I tried to install the drivers one time (everything went down, PineAp crashed and the interfaces went crazy) and the second time didn't work. Any advice or maybe in the near future will be supported.

**Reply by xchwarze** [reply]
Here is the list of drivers that work 100% well
https://github.com/xchwarze/wifi-pineapple-cloner/discussions/1

**Reply by tuconnaisyouknow** [reply]
> I don't know what chipset that one uses but I have the AWUS036ACM and it works perfectly

Hi, sorry for answering this late to your comment but I have a question. I have the AWUS036ACM with the GL-AR750S-Ext too and I resetted the Pineapple to factory settings with the dongle plugged in. The 5GHz Recon works perfectly but the 2.4GHz doesn't work (it doesn't display any network). Is it normal and if not do you know how to solve it ?
